.travis.yml

```diff
@@ -16,7 +16,8 @@ env:
   global:
     - CCACHE_CPP2=yes
   matrix:
-    - PARALLEL=yes
+    - PARALLEL=ON
+    - PARALLEL=OFF
 
 script:
   - >
```

Comment:
Btw, with using `find_package(MPI)`, there is no need to specify the compiler directly here anymore.